locallibs/install.py

```diff
@@ -87,6 +88,18 @@ def install_extras(framework_path, version="2.7", requirements_file=None,
                    install_wheel=False, upgrade_pip=False):
     """install all extra pkgs into Python framework path"""
     print()
+    python_guard_path = expanduser("~/Library/Python/%s") % version
+    if os.path.exists(python_guard_path):
+        print('*********************************************************')
+        print('*** Python user files exist that conflict with the    ***')
+        print('*** version of relocatable python you are trying to   ***')
+        print('*** create. This will likely result in site-packages  ***')
```

Comment:
"This can result in extra packages to be added not being installed properly" might be more accurate.